manifests/t/The0xGingi-BrowserAuthors/0xGingi-BrowserInstaller/114.0.5735.110/The0xGingi-BrowserAuthors.0xGingi-BrowserInstaller.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: The0xGingi-BrowserAuthors.0xGingi-BrowserInstaller
+PackageVersion: 114.0.5735.110
+PackageLocale: en-US
+Publisher: The 0xGingi-Browser Authors
+PackageName: 0xGingi-Browser Installer

```

```suggestion
PackageName: 0xGingi-Browser
```
This should match control panel